d2graph/d2graph.go

```diff
@@ -131,6 +133,22 @@ type Attributes struct {
 	Constraint Scalar `json:"constraint"`
 }
 
+func (a *Attributes) ApplyTextTransform() {
+	if a.Style.NoTextTransform() {
+		return
+	}
+
+	if a.Style.UppercaseTextTransform() {
+		a.Label.Value = strings.ToUpper(a.Label.Value)
+	}
+	if a.Style.LowercaseTextTransform() {
+		a.Label.Value = strings.ToLower(a.Label.Value)
+	}
+	if a.Style.CapitalizeTextTransform() {
+		a.Label.Value = cases.Title(language.Und).String(a.Label.Value)
+	}
```

Comment:
inline the method calls please. no need to jump around for a one-liner used in one place